pkgs/by-name/se/seabios/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchgit
+, acpica-tools
+, python3
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "seabios";
+  version = "1.16.3";
+
+  src = fetchgit {
+    url = "https://git.seabios.org/seabios.git";
+    rev = "rel-${finalAttrs.version}";
+    hash = "sha256-hWemj83cxdY8p+Jhkh5GcPvI0Sy5aKYZJCsKDjHTUUk=";
+  };
+
+  outputs = [ "out" "doc" ];
+
+  nativeBuildInputs = [ python3 ];
+
+  buildInputs = [ acpica-tools ];
+
+  strictDeps = true;
+
+  hardeningDisable = [ "pic" "stackprotector" "fortify" ];
+
+  postConfigure = ''
+    cp ${./config.txt} .config
+    make olddefconfig
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -pv $doc/share/doc/seabios-${finalAttrs.version}/
+    cp -v docs/* $doc/share/doc/seabios-${finalAttrs.version}/
+    install -Dm644 out/Csm16.bin -t $out/share/seabios/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "http://www.seabios.org";
+    description = "Open source implementation of a 16bit x86 BIOS";
+    longDescription = ''
+      SeaBIOS is an open source implementation of a 16bit x86 BIOS.
+      It can run in an emulator or it can run natively on x86 hardware with the
+      use of coreboot.
+    '';
+    license = lib.licenses.lgpl3;
+    maintainers = with lib.maintainers; [ AndersonTorres ];
+    platforms = with lib.platforms;
+      lib.intersectLists (unix) (x86);
```

Comment:
I think we use inspect patterns rather than intersection for this type of stuff.

Reply:
I don't know about them. I have seen them once while reviewing some code from Alyssais, but I didn't understand.